src/fsharp/FSharp.Core.nuget/FSharp.Core.LatestNuget.nuspec

```diff
@@ -6,7 +6,7 @@
         <title>FSharp.Core for F# 4.5</title>
         <summary>FSharp.Core for F# 4.5</summary>
         <description>
-            FSharp.Core redistributables from Visual F# Tools version 10.2 For F# 4.5
+            FSharp.Core redistributables from Visual F# Tools version 10.3 For F# 4.5
```

Comment:
All other instances of this number are `10.2.3`; is this `10.3` intentional?

Reply:
No.  It should have stayed at 10.2   good eyes.